lntest/harness.go

```diff
@@ -342,7 +342,6 @@ func (h *HarnessTest) SetupStandbyNodes() {
 	// above a good number of confirmations.
 	const totalTxes = 200
 	h.MineBlocksAndAssertNumTxes(numBlocksSendOutput, totalTxes)
-	h.MineBlocks(numBlocksSendOutput)
```

Comment:
Hmm I don't really understand the comment of the sweeper here, because I don't think the sweeper is involved here in general, we are using the `SendRawTransaction` of the miner imo, but maybe I misunderstand something here.

Reply:
yeah my bad - it's from another PR, removed the commit msg!